### PR TITLE
Added bestsellers' items

### PR DESCRIPTION
**Pull request by Meethiravi**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -382,3 +382,161 @@ section.padded {
...
+  padding-top: 32px;
+  padding-bottom: 32px;
+}
+
+.bestsellers {
+  display: flex;
+  justify-content: space-between;
+  flex-flow: row wrap;
+}
+.bestsellers::after {
```
Comment (kushagrarora17) [inline css/styles.css:425]: Why is it required?

File: css/styles.css [hunk 2]
```diff
@@ -382,3 +382,161 @@ section.padded {
...
+}
+
+.star-widget > input:checked + label:hover,
+.star-widget > input:checked ~ label:hover,
+.star-widget > label:hover ~ input:checked ~ label,
+.star-widget > input:checked ~ label:hover ~ label {
+  color: #ffed85;
+} */
+
+input:not(:checked) ~ label:hover,
```
Comment (kushagrarora17) [inline css/styles.css]: this style gets implemented for all input and label tags, which is not expected. Use classes.

File: css/styles.css [hunk 3]
```diff
@@ -382,3 +382,161 @@ section.padded {
...
+  height: 40px;
+  padding: 6px;
+  cursor: pointer;
+  background-color: #fff;
+}
+
+.product-btn:hover {
+  background-color: #222;
+  border: none;
+  transition: background-color 400ms linear;
```
Comment (kushagrarora17) [inline css/styles.css]: transition property should be defined in the class and not in pseudo-class (:hover)

File: css/utils.css [hunk 4]
```diff
@@ -13,4 +13,9 @@
 .img-responsive {
   width: 100%;
   max-height: auto;
+}
+
+.img-responsive2 {
```
Comment (kushagrarora17) [inline css/utils.css]: use meaningful class names - "img-responsive-fit"

File: css/styles.css (hunk 1 above)
Comment (Meethiravi) [reply css/styles.css:425]: This is for second row elements spacing. Without this the last two items are not spaced properly.

File: index.html [hunk 5]
```diff
@@ -205,6 +205,166 @@ <h2 class="banner-title">
...
+          <figure>
+            <img
+              src="/images/heart-hand-warmer.jpeg"
+              alt="heart hand warmer"
+              class="img-responsive2"
+            />
+          </figure>
+          <h4 class="item-name">Heart Hand Warmer</h4>
+          <h4 class="item-price">$5.99</h4>  
+          <div class="star-widget">
```
Comment (kushagrarora17) [inline index.html]: this kind of solution is desired for star widget: https://codepen.io/hesguru/pen/BaybqXv

**Pull request merged**